deploy/jenkins/jenkins-server-setup.sh

```diff
@@ -18,7 +18,10 @@ apt-get update
 apt-get install -y jenkins=2.263.4
 
 echo -e "\n\e[0;32m${bold}Installating PIP${normal}"
-apt-get install -y python-pip
+udo apt update 
```

Comment:
s missing from sudo

Reply:
@rjshrjndrn Ok. I am correcting this now.